libraries/rib-base/src/main/java/com/badoo/ribs/core/Node.kt

```diff
@@ -160,8 +160,10 @@ open class Node<V : RibView> @VisibleForTesting internal constructor(
             view?.let { view ->
                 view.androidView.restoreHierarchyState(savedViewState)
                 lifecycleManager.onViewCreated()
+                val lifecycle = lifecycleManager.viewLifecycle!!.lifecycle
+                (view as? RibView.LifecycleAware)?.onCreate(lifecycle)
```

Comment:
Shouldn't we here just add it to list of plugins, and notify all plugins as a next step?

Reply:
Do not want to make list of Plugins mutable. If we add the view to plugin list, then we also need to remove it when destroyed.